OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -1319,7 +1322,7 @@ public void run() {
          return;
       }
 
-      if (getRemoteParams().useEmailAuth && emailAuthHash == null) {
+      if (getRemoteParams() != null && getRemoteParams().useEmailAuth && emailAuthHash == null) {
```

Comment:
remote params shouldn't be null at this point, because of the shouldQueueTaskForInit check. Why is it needed?

Reply:
When there is a connection issue and the remote configuration is not previously setup, then `getRemoteParams()` is Null and this is the breaking point for the issue found.

Reply:
Sry I might be missing some part, but shouldQueueTaskForInit check for remote == null, if remote is still null because the network failed, then the task should be queue again, or why is it not being queue again?

Reply:
It does get queued, but it also continues evaluating the statements below. Another approach could be a return statement within the task schedule block, what do you think?

Reply:
oooh I see, that method is missing the return! good catch, so that is why is failing

Reply:
No other changes should be needed but the return stmt

Reply:
compare with the other methods using shouldQueueTaskForInit

Reply:
This is redundant code, the `shouldQueueTaskForInit` check and the `return` added above would cover this so I think we should remove the `null` check that was added in this PR.